cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -19,6 +24,53 @@ func Start(cfg config.LocalConfig) (exitCode int) {
 	logger := supportlog.New()
 	logger.SetLevel(cfg.LogLevel)
 
+	httpPortUint := uint(cfg.CaptiveCoreHTTPPort)
+	var captiveCoreTomlParams ledgerbackend.CaptiveCoreTomlParams
+	captiveCoreTomlParams.HTTPPort = &httpPortUint
+	captiveCoreTomlParams.HistoryArchiveURLs = cfg.HistoryArchiveURLs
+	captiveCoreTomlParams.NetworkPassphrase = cfg.NetworkPassphrase
+	captiveCoreTomlParams.Strict = true
+	captiveCoreToml, err := ledgerbackend.NewCaptiveCoreTomlFromFile(cfg.CaptiveCoreConfigPath, captiveCoreTomlParams)
```

Comment:
@SirTyson , has [a PR merging soon in go repo](https://github.com/stellar/go/pull/4733/files#diff-c7702ef819e52917ed3868588dff5273774c61a6aa8d36c1644d7ba251fb7756) which will config captive core to use BucketListDB by default, no change needed here, just wanted to mention it, as once it's in, might help in some of the spin up time.

Reply:
Ah, fantastic. I will bump the PR and check the spin up time!